lms/static/sass/base/_variables.scss

```diff
@@ -306,6 +306,10 @@ $credit-color-base: rgb(244,195,0); // accessible with black text
 // edx-specific: Studio/Staff actions
 $staff-color: $pink;
 
+// from the edX Pattern Library
```

Comment:
The pattern library doesn't play well with others (i.e. a page has to be either all pattern library or all the old styles. There is a later story to  move this page to all Pattern Library, but that is blocked by the lack of a Pattern Library footer).